src/v3/src/components/StepperLink/StepperLink.tsx

```diff
@@ -43,23 +44,15 @@ const StepperLink: UISchemaElementComponent<{
   };
 
   return (
-    <LinkMui
-      component="button"
-      variant="body1"
-      role="link"
+    // eslint-disable-next-line jsx-a11y/anchor-is-valid
```

Comment:
This PR should have been looped into my migration of `Link`, it's nearly identical